contracts/shared/libraries/SafeMath.sol

```diff
@@ -0,0 +1,17 @@
+pragma solidity =0.6.6;
```

Comment:
if you upgrade to sol 0.8 there's no need to include this as is part of solidity. Also, why write your own instead of including OZ directly?